WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptView.swift

```diff
@@ -15,6 +16,14 @@ struct POSSendReceiptView: View {
         EmailFormatValidator.validate(string: textFieldInput)
     }
 
+    private var keyboardHeight: CGFloat {
+        if keyboardObserver.keyboardHeight < Constants.keyboardShownButtonSpacing {
```

Comment:
I don't really understand the intent of this comparison... could you explain a little?

Why would the keyboard being taller than the shown button spacing mean that we should return the shown button spacing? Surely there's some other threshold to decide that?

At the moment, there are some unexpected changes of this spacing at times, and it might be because of this comparison...